package.json

```diff
@@ -66,7 +66,7 @@
     "typeahead.js": "^0.10.5",
     "url-loader": "^0.5.5",
     "uuid": "^2.0.1",
-    "webpack": "^1.7.2",
+    "webpack": "1.13.0",
```

Comment:
The latest version is 1.13.1, which ^1.7.2 happily matches to. 1.13.1 will not build our assets because of some Dependency dependency error.